application-github-statistics-ui/src/main/resources/GitHubStats/CommittersMacro.xml

```diff
@@ -230,13 +385,14 @@ rkJggg==
   #if ($isCommitter || (!$isCommitter &amp;&amp; "$!xcontext.macro.params.contributors" == "true"))
     #set ($avatar = $authorData.avatar)
     #if ("$!avatar" == '')
-      #set ($avatarDisplay = "(% style='width:85px' %)[[image:attach:GitHubStats.CommittersMacro@unknown.png||width='80' height='80']]")
+      #set ($avatarImg = "attach:GitHubStats.CommittersMacro@unknown.png")
```

Comment:
I think we should could call it "avatarReference" (see http://platform.xwiki.org/xwiki/bin/view/Main/XWikiSyntax?syntax=2.1&section=Images).